src/createServer.js

```diff
@@ -1,3 +1,32 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+
+const http = require('http');
+const { checkOnError } = require('./checkOnError');
+const { createResponseBody } = require('./createResponseBody');
+const { toSplitUrl } = require('./splitUrl');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const queryString = req.url.split('?');
+    const [text, targetCase] = toSplitUrl(queryString);
```

Comment:
It's better to use destructuring assignment with the 'toSplitUrl' function result, rather than using array indexes.

Reply:
I didn't get it.